lib/normalize_grammar.re

```diff
@@ -1,5 +1,78 @@
 open Common;
 
-let normalize = _ast =>
-  raise (Todo);
-  
\ No newline at end of file
+let normalize_to_simple = body => {
```

Comment:
You can use module aliases to be less verbose:
module A = Ast_grammar;
module B = Ast_grammar_normalized;

so you can then just write A.TOKEN => ... B.TOKEN which is shorter.

Reply:
Ok.